dotnet/src/Experimental/Process.Server/Process.Server.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Process.Server</AssemblyName>
```

Comment:
I wonder if `Microsoft.SemanticKernel.Process.Local` or `Microsoft.SemanticKernel.Process.LocalRuntime` would be a better name?

This can be changed before we merge to main.